terraform.yml

```diff
@@ -1,13 +1,13 @@
-image:
-  name: hashicorp/terraform:0.12.9
-  entrypoint: [""]
```

Comment:
@gadiener you sure this can be taken away?

Reply:
oh yes